src/styles/index.scss

```diff
@@ -1,3 +1,89 @@
+$font-family: Arial, sans-serif;
+$font-size: 30px;
+$cell-color: #eee;
+$cell-size: 100px;
+$border-size: 1px;
+$gap-size: 1px;
+$cell-plus-border-size: $cell-size + $border-size;
+
+* {
+  box-sizing: border-box;
+  font-family: $font-family;
+  font-size: $font-size;
+}
+
+ul li {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
```

Comment:
better combine this styles to avoid copy/past:

body,
ul,
li {
  margin: 0;
  padding: 0;
}